src/theme/github_dark.css.js

```diff
@@ -0,0 +1,146 @@
+  }
+  
+  .ace-github-dark .ace_marker-layer .ace_step {
+    background: rgb(127, 111, 19)
+  }
+  
+  .ace-github-dark .ace_marker-layer .ace_bracket {
+    margin: -1px 0 0 -1px;
+    border: 1px solid #404F7D
+  }
+  
+  .ace-github-dark .ace_marker-layer .ace_active-line {
+    background: #00346E
+  }
+  
+  .ace-github-dark .ace_gutter-active-line {
+    background-color: #24292e
+  }
+  
+  .ace-github-dark .ace_marker-layer .ace_selected-word {
+    border: 1px solid #003F8E
+  }
+  
+  .ace-github-dark .ace_invisible {
+    color: #404F7D
+  }
+  
+  .ace-github-dark .ace_keyword,
+  .ace-github-dark .ace_meta,
+  .ace-github-dark .ace_storage,
+  .ace-github-dark .ace_storage.ace_type,
+  .ace-github-dark .ace_support.ace_type {
+    color: #ff7b72
+  }
+  
+  .ace-github-dark .ace_keyword.ace_operator {
+    color: #79c0ff
+  }
+  
+  .ace-github-dark .ace_constant.ace_character,
+  .ace-github-dark .ace_constant.ace_language,
+  .ace-github-dark .ace_constant.ace_numeric,
+  .ace-github-dark .ace_keyword.ace_other.ace_unit,
+  .ace-github-dark .ace_support.ace_constant,
+  .ace-github-dark .ace_variable.ace_parameter {
+    color: #FFC58F
+  }
+  
+  .ace-github-dark .ace_invalid {
+    color: #FFFFFF;
+    background-color: #F99DA5
+  }
+  
+  .ace-github-dark .ace_invalid.ace_deprecated {
+    color: #FFFFFF;
+    background-color: #ff7b72
+  }
+  
+  .ace-github-dark .ace_fold {
+    background-color: #BBDAFF;
+    border-color: #FFFFFF
+  }
+  
+  .ace-github-dark .ace_entity.ace_name.ace_function,
+  .ace-github-dark .ace_support.ace_function,
+  .ace-github-dark .ace_variable {
+    color: #BBDAFF
+  }
+  
+  .ace-github-dark .ace_support.ace_class,
+  .ace-github-dark .ace_support.ace_type {
+    color: #FFEEAD
+  }
+  
+  .ace-github-dark .ace_heading,
+  .ace-github-dark .ace_markup.ace_heading,
+  .ace-github-dark .ace_string {
+    color: #9fcef6
+  }
+  
+  .ace-github-dark .ace_entity.ace_name.ace_tag,
+  .ace-github-dark .ace_entity.ace_other.ace_attribute-name,
+  .ace-github-dark .ace_meta.ace_tag,
+  .ace-github-dark .ace_string.ace_regexp,
+  .ace-github-dark .ace_variable {
+    color: #FF9DA4
+  }
+  
+  .ace-github-dark .ace_comment {
+    color: #7285B7
+  }
+  
+  .ace-github-dark .ace_indent-guide {
+    background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAACCAYAAACZgbYnAAAAEklEQVQImWNgYGBgYJDzqfwPAANXAeNsiA+ZAAAAAElFTkSuQmCC) right repeat-y
+  }
+  
+  .ace-github-dark .ace_indent-guide-active {
+    background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAACCAYAAACZgbYnAAAAEklEQVQIW2PQ1dX9zzBz5sz/ABCcBFFentLlAAAAAElFTkSuQmCC) right repeat-y;
+  }
+
+  .ace-github .ace_constant.ace_buildin {
+    color: #0086B3;
+  }
+  
+  .ace-github .ace_support.ace_function {
+    color: #0086B3;
+  }
+  
+  .ace-github .ace_comment {
+    color: #90bbe1;
+    font-style: italic;
+  }
+  
+  .ace-github .ace_variable.ace_language  {
+    color: #ffffff;
+  }
```

Comment:
These lines here are for `ace-github` and not `ace-github-dark` and they seem to be duplicates from above, I think you forgot to remove these?